keyboards/preonic/keymaps/kyrias/config.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2015-2021 Jack Humbert
+ * Copyright 2022-2023 Johannes Löthberg (@kyrias)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifndef __ASSEMBLER__
```

Comment:
this should be moved to the keymap.c

Reply:
That would require moving the `TRI_LAYER_*_LAYER` defines to `keymap.c` as well though?

Reply:
Ping?

It seems very wrong to me to require the `TRI_LAYER_*_LAYER` config defines to be placed in `keymap.c`.

Reply:
Yes, it wouldn't work. Just don't use the enum entries for them.

Reply:
What's even the point of using enums in some places if we're going to need to manually specify integers in others?  :/

Reply:
because enums in the config.h can cause issues.  And they're mostly just placeholders for readability/reference.

But if you want to use the names,  you could do this in your keymap.c: 
```c
void keyboard_post_init_user(void) {
    set_tri_layer_lower_layer(_LOWER);
    set_tri_layer_upper_layer(_UPPER);
    set_tri_layer_adjust_layer(_ADJUST);
}
```

Reply:
What kind of issues?  I switched to using `#ifndef __ASSEMBLER__` because I saw that `keyboards/handwired/promethium/config.h` does it that way, so I'm curious what kind of issues this could cause that isn't already an issue?

I want to use them to ensure correctness, not readability.  But I can certainly switch to doing it on init.